docs/user/authentication-overview.md

```diff
@@ -0,0 +1,134 @@
+# Authentication Overview
+
+This guide provides an overview of authentication in Flight Control to help you understand the available options and choose the right authentication method for your deployment.
+
+## Default Authentication by Deployment
+
+Flight Control automatically configures authentication based on your deployment environment:
+
+| Deployment Environment | Default Authentication | Description |
+|------------------------|----------------------|-------------|
+| **AAP (Ansible Automation Platform)** | AAP Gateway | Integrates with AAP Gateway for authentication and authorization |
+| **OpenShift** | OpenShift OAuth | Uses OpenShift's OAuth server for authentication |
+| **Standalone (Quadlet/Podman)** | OIDC | Flight Control API server uses OIDC authentication. [PAM Issuer](pam-authentication.md) (bundled OIDC provider) is deployed by default |
+| **Kubernetes (non-OpenShift)** | Kubernetes RBAC | Uses Kubernetes service accounts and RBAC for authentication |
+
+### Supported Authentication Methods
+
+Flight Control API server supports the following authentication methods:
+
+#### 1. OIDC (OpenID Connect)
+
+Standard OpenID Connect protocol. Works with any OIDC-compliant provider (Azure AD, Okta, Keycloak, Google, etc.). Supports dynamic provider configuration, flexible organization and role mapping, and multiple simultaneous providers.
```

Comment:
We haven't tested this (or have we?), so should probably say "should work" or even say "tested with Keycloak and X".